Formula/s/sbt.rb

```diff
@@ -6,9 +6,12 @@ class Sbt < Formula
   sha256 "32c15233c636c233ee25a2c31879049db7021cfef70807c187515c39b96b0fe6"
   license "Apache-2.0"
 
+  # Upstream creates releases that use a stable tag (e.g., `v1.2.3`) but are
+  # labeled as "pre-release" on GitHub before the version is released, so it's
+  # necessary to use the `GithubLatest` strategy.
```

Comment:
```suggestion
  # Upstream sometimes creates releases that use a stable tag (e.g., `v1.2.3`)
  # but are labeled as "pre-release" on GitHub.
```

This comment is intended for when upstream's release process involves them marking a version as pre-release and removing the pre-release label when it's officially released. 1.10.8 will presumably remain as "pre-release" and this looks like a one-off issue for this repository, so this comment doesn't apply to this situation.

The suggestion above adapts the comment so it makes sense in this situation and I dropped the trailing text about the `GithubLatest` strategy since the `stable` URL is a GitHub release asset.